CustomSortStrings.py

```diff
@@ -0,0 +1,30 @@
+"""
+Time Complexity : O(m+n) - where m is len(S) and n is the len(T)
+Space Complexity : O(n)
+Did this code successfully run on Leetcode : Yes
```

Comment:
Space complexity will be constant as there'll be 26 characters in map at maximum.